Splunk_TA_paloalto/default/props.conf

```diff
@@ -448,3 +448,13 @@ LOOKUP-vendor_info_for_pan_aperture = pan_vendor_info_lookup sourcetype OUTPUT v
 category = Network & Security
 SHOULD_LINEMERGE = 0
 pulldown_type = 1
+
+[pan_decryption]
+rename = pan:decryption
+
+[pan:decryption]
+SHOULD_LINEMERGE = false
+TIME_PREFIX = ^(?:[^,]*,){6}
+MAX_TIMESTAMP_LOOKAHEAD = 32
+
+REPORT-search = extract_decryption
```

Comment:
I don't see an field aliasing for CIM compliance here. Is any needed?

Reply:
None is needed at this time